tests/e2e/crash-before-createdb/10-assert.yaml

```diff
@@ -11,12 +11,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: vertica.com/v1beta1
```

Comment:
Sorry if I caused confusion here, but can we keep the old check too?  So it would check each of the 3 pods, plus the VerticaDB.

Reply:
Oh ok. Will do.